functions/attendees-api/handler/handler.go

```diff
@@ -47,6 +48,12 @@ func (h Handler) HandleRequest(ctx context.Context, request events.APIGatewayPro
 
 }
 
+func (h Handler) getAuthCodeFromPath(path string) string {
+	r := regexp.MustCompile("/attendees/([A-Za-z0-9]{6})")
```

Comment:
Not sure if regex is neccessary? Can use the split function on the path string

Reply:
you could true.  I wanted to use it to also validate the parameter somewhat (he says making up a reason why he didn't use split)